test/tools/ossfuzz/AbiV2IsabelleFuzzer.cpp

```diff
@@ -58,8 +58,9 @@ DEFINE_PROTO_FUZZER(Contract const& _contract)
 		// We target the default EVM which is the latest
 		langutil::EVMVersion version;
 		EVMHost hostContext(version, evmone);
-		string contractName = ":C";
-		CompilerInput cInput(version, {{"test.sol", contractSource}}, contractName, OptimiserSettings::minimal(), {});
+		string contractName = "C";
```

Comment:
Why is the colon not needed here? I see that `contractName` is passed to `CompilerStack` and compiler stack identifies contracts with `<file name>:<contract name>` so even if `<file name>` is empty, the colon should still be there.

My assumption would be that some piece of code in `CompilerInput` or a related class is responsible concatenating the file name and contract name and inserts the colon but I do not see any such code (I did not look too deeply though).

Reply:
I think this issue (not removing the colon) should have been done in an earlier PR where the utility class was refactored but overlooked.

I use contract name the same way as in here for example https://github.com/ethereum/solidity/blob/f188f3d9ed261223e16bcacff0d55c30e94500db/test/libsolidity/GasCosts.cpp#L197

Reply:
So I'm curious why this works. I remember that when writing tests for `FunctionCallGraph` the colon was always there in the name...

But well, if it works I guess it must be ok.